client/components/account-balances/index.tsx

```diff
@@ -130,129 +131,135 @@ const AccountBalances: React.FC = () => {
-																target="_blank"
-																href={
-																	documentationUrls.negativeBalance
-																}
-															/>
-														),
-													},
-											  } )
-											: interpolateComponents( {
-													mixedString: __(
-														'The amount of funds available to be deposited. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
-														'woocommerce-payments'
-													),
-													components: {
-														learnMoreLink: (
-															// eslint-disable-next-line jsx-a11y/anchor-has-content
-															<a
-																rel="external noopener noreferrer"
-																target="_blank"
-																href={
-																	documentationUrls.depositSchedule
-																}
-															/>
-														),
-													},
-											  } )
-									}
-								/>
-							}
-						/>
-						<BalanceBlock
-							id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
-							title={ fundLabelStrings.pending }
-							amount={ tab.pendingFunds }
-							currencyCode={ tab.currencyCode }
-							tooltip={
-								<BalanceTooltip
-									label={ `${ fundLabelStrings.pending } tooltip` }
-									content={
-										tab.pendingFunds < 0
-											? interpolateComponents( {
-													mixedString: __(
-														'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
-														'woocommerce-payments'
-													),
-													components: {
-														learnMoreLink: (
-															// eslint-disable-next-line jsx-a11y/anchor-has-content
-															<a
-																rel="external noopener noreferrer"
-																target="_blank"
-																href={
-																	documentationUrls.negativeBalance
-																}
-															/>
-														),
-													},
-											  } )
-											: interpolateComponents( {
-													mixedString: sprintf(
-														// Translators: %d is the number of days, e.g. 1, 2, 3, etc.
-														__(
-															'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
-															'woocommerce-payments'
-														),
-														tab.delayDays
-													),
-													components: {
-														learnMoreLink: (
-															// eslint-disable-next-line jsx-a11y/anchor-has-content
-															<a
-																rel="external noopener noreferrer"
-																target="_blank"
-																href={
-																	documentationUrls.depositSchedule
-																}
-															/>
-														),
-													},
-											  } )
-									}
-								/>
-							}
-						/>
-					</Flex>
-					{ tab.instantBalance && tab.instantBalance.amount > 0 && (
+			{ ( tab: BalanceTabProps ) => {
+				const totalBalance = tab.availableFunds + tab.pendingFunds;
+
+				const negativeBalanceTooltipContent = interpolateComponents( {
```

Comment:
I assign the content of tooltip when balance is negative to a variable here because it's going to be used for total balance and available balance when either of them is negative.

Reply:
The linked documentation, and the tooltip really only apply to total balance. 

We should carefully consider how we present the special-case negative balance tooltip.  It might need input from design. 

Here's my take:

- The (new) tooltips for total balance and available balance are always true and relevant. When balance is negative, that doesn't override the "total balance" tooltip.
- The negative balance info is a special case, and is most relevant for "total balance" – if total balance is persistently negative, we will withdraw funds from merchant account.

So I think we have two options:

- Don't show a special tooltip for negative balance (i.e. revert this behaviour, because our new tooltips are clear enough).
- OR
- When the merchant has a negative total balance, add the info about negative balance to the "total balance" tooltip.

Keen for thoughts from @rogermattic and @csmcneill …